Dockerfile

```diff
@@ -26,31 +28,23 @@ LABEL maintainer="Plone Community <dev@plone.org>" \
       org.label-schema.description="Plone backend image image using Python 3.9" \
       org.label-schema.vendor="Plone Foundation"
 
-COPY --from=builder /wheelhouse /wheelhouse
-
-RUN useradd --system -m -d /app -U -u 500 plone \
-    && runDeps="git libjpeg62 libopenjp2-7 libpq5 libtiff5 libxml2 libxslt1.1 lynx netcat poppler-utils rsync wv busybox gosu libmagic1 make" \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends $runDeps \
-    && busybox --install -s \
-    && rm -rf /var/lib/apt/lists/* /usr/share/doc \
-    && mkdir -p /data/filestorage /data/blobstorage /data/log /data/cache
+COPY --from=builder --chown=500:500 /venv /app
+COPY --chown=500:500 skeleton/* /app/
+
+RUN <<EOT
+    ls -la /app
+    useradd --system -m -d /app -U -u 500 plone
+    runDeps="git libjpeg62 libopenjp2-7 libpq5 libtiff5 libxml2 libxslt1.1 lynx netcat poppler-utils rsync wv busybox gosu libmagic1 make"
+    apt-get update
+    apt-get install -y --no-install-recommends $runDeps
+    busybox --install -s
+    rm -rf /var/lib/apt/lists/* /usr/share/doc
+    mkdir -p /data/filestorage /data/blobstorage /data/log /data/cache
+    chown -R /data plone:plone
+    ln -s /data var
+EOT
 
 WORKDIR /app
-
-RUN python -m venv . \
-    && ./bin/pip install -U "pip==${PIP_VERSION}" \
-    && ./bin/pip install --force-reinstall --no-index --no-deps ${PIP_PARAMS} /wheelhouse/* \
-    && ./bin/pip install "relstorage==3.4.5" "psycopg2==2.9.3" ${PIP_PARAMS} \
```

Comment:
We still need relstorage and psycopg2 to be installed somewhere, right?

(Also, maybe we can use psycopg2-binary?)

Reply:
Oh, I just found them in EXTRA_PACKAGES